packages/landing/static/index.css

```diff
@@ -167,6 +167,10 @@ button {
     transition: right 0.5s;
 }
 
+#side-nav>h3 {
```

Comment:
This seems to be a different format to everything else, do we have a css formatter somewhere

Reply:
fixed the indentation